verilog/analysis/verilog_linter_configuration_test.cc

```diff
@@ -775,5 +775,76 @@ TEST(RuleBundleTest, ParseRuleBundleSkipComments) {
   }
 }
 
+// ConfigureFromFlags Tests
+TEST(ConfigureFromFlagsTest, Basic) {
+  LinterConfiguration config;
+
+  LinterConfigurationFlags flags = {
+      RuleSet::kDefault,
+      RuleBundle(),
+      "-",
+      false,
+      false,
+      "filename"
+      "filename"};
+
+  auto status = config.ConfigureFromFlags(flags);
+  EXPECT_TRUE(status.ok());
+}
+
+TEST(ConfigureFromFlagsTest, RulesNumber) {
+  LinterConfiguration config;
+
+  LinterConfigurationFlags flags = {
+      RuleSet::kAll,
+      RuleBundle(),
+      "-",
+      false,
+      false,
+      "filename"
+      "filename"};
```

Comment:
So a lot of unnamed parameters are a little bit hard to follow as we don't see the definition of LinterConfigurationFlags side-by-side. If the structs would only contain 2-3 parameters, {}-initiaalization would be fine.

Until C++20 catches up with C-89 and allows designated initializers (C++, why oh why arent they there?), we probably have to do manual init:
```
  flags.config_file = "-"
  flags.is_custom = false;
  // ...
```

I suspect that will also clash with the use of const references in the struct, but discussed above.

Reply:
I think in our case we can use the designated initializes as we are initializing everything anyway. Thanks for the tip!

Reply:
Keep in mind, the current minimum standard for building is `-std=c++11`.